### DISCUSSION TITLE
Include content of other page

### DISCUSSION THREAD

**Discussion by codewing** [dotnet/docfx#8026] [Q&A] [answered]
Similar to creating a link to another page is it possible to include the content of another page?

something like

```markdown

# Header

The concept works like this:

{{<xref:id_of_another_file>}}

And we build on top of that...

```

## Why?

We write documentation of general concepts and would like to include these definitions/pages in other sites as well.

**Comment by KalleOlaviNiemitalo** [comment, marked as answer]
yes, see [File Inclusion](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html#file-inclusion)
